### DISCUSSION TITLE
Blazorise Grid to CSV

### DISCUSSION THREAD

**Discussion by DavidCountry** [Megabit/Blazorise#2615] [General]
Hello,
Is there a built in process to take the contents from the DataGrid and export to a downloadable CSV?

**Comment by David-Moreira** [comment]
No. But it shouldn't be hard for you to implement a new Button on ButtonRow that grabs the data and sends it into a third party csv specialised library

However we could consider those as extensions to Blazorise.Datagrid
Something like 
Blazorise.Datagrid.Export.CSV
Blazorise.Datagrid.Export.Excel
But most likely we would use third party libraries ourselves... No point in reinventing the wheel...
What do you think @stsrki?

**Reply by stsrki** [reply]
I agree. No need to make it part of the core library. So having a separate project like Blazorise.Datagrid.Export.CSV or Blazorise.Datagrid.Export.Excel makes more sense. We only need to provide an API to read the data.

**Reply by David-Moreira** [reply]
If you agree, then might as well create an issue/feature, to track this, maybe for 0.9.5 shouldn't be too hard theoretically.
